Codecademy/gamut
# fix: DataGrid reformatted to `table` and aria-sort added

### Overview


Refactors the DataTable + DataList components to properly be semantic HTML tables and adds `aria-sort` to the appropriate controls


### PR Checklist

- [x] Related to JIRA ticket: [GM-945]

#### Testing Instructions


1. Navigate to the DataList
2. Test that it still works as intended (filter, select, sort actions specifically)
3. Turn on VO and see that you can navigate it as a table (instructions [here](https://a11y-101.com/development/tables)
4. You can also inspect to see no errors and in console and that it is the correct table structure
5. Go to DataTable 
6. Test that it still works as intended (filter, sort actions specifically)
7. Turn on VO and see that you can navigate it as a table ((instructions [here](https://a11y-101.com/development/tables))
8. Go to List and see that they look the same as before

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/39068)  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/9058)  | [Portal Env](http://www.google.fr/ 'Named link title')   |





[GM-945]: https://skillsoftdev.atlassian.net/browse/GM-945?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
